platforms/windows/SQLitePlugin/SQLitePlugin.h

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include "pch.h"
+
+using namespace winrt;
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Storage;
+
+namespace SQLitePlugin
+{
+    /* 
+    * SQLite Plugin for React Native. 
+    * 
+    * As of now, it supports callbacks only.
+    * All operations on the database have to be serialized to avoid race conditions.
+    * Each operation posts a task to SerialReactDispatcher 
+    * which executes them sequentially in the background.
+    */
+
+    REACT_STRUCT(CloseOptions);
+    struct CloseOptions
+    {
+        REACT_FIELD(Path, L"path");
+        // Path at which the database is located
+        std::string Path;
+    };
+
+    REACT_STRUCT(DeleteOptions);
+    struct DeleteOptions
+    {
+        REACT_FIELD(Path, L"path");
+        // Path at which the database is located
+        std::string Path;
+    };
+
+    REACT_STRUCT(DBArgs);
+    struct DBArgs
+    {
+        REACT_FIELD(DBName, L"dbname");
+        std::string DBName;
+    };
+
+    REACT_STRUCT(DBQuery);
+    struct DBQuery
+    {
+        REACT_FIELD(QID, L"qid");
+        int QID;
+
+        REACT_FIELD(Params, L"params");
+        JSValueArray Params; // optional
+
+        REACT_FIELD(SQL, L"sql");
+        std::string SQL;
+    };
+
+    REACT_STRUCT(EchoStringValueOptions);
+    struct EchoStringValueOptions
+    {
+        REACT_FIELD(Value, L"value");
+        std::string Value;
+    };
+
+    REACT_STRUCT(ExecuteSqlBatchOptions);
+    struct ExecuteSqlBatchOptions
+    {
+        REACT_FIELD(DBArgs, L"dbargs");
+        DBArgs DBArgs;
+        REACT_FIELD(Executes, L"executes");

```

Comment:
Unless the name "executes" needs to match other platforms, I'd change it to sth else. Why not 'queries'?

Reply:
My understanding that it uses that name to parse the arguments from JSON that is passed in. So the name needs to match whatever is in JS and on other platforms.

Reply:
ok